topologies/service_tags/generate_topology.py

```diff
@@ -18,9 +18,15 @@ def generateCallGraph(serviceRoutes, numServices, fanOut, maxDepth):
         for (service, routes) in serviceRoutes.items():
             routesJson = []
             for route in routes:
+                clientSpanCallsJson = {}
                 downStreamCallsJson = {}
                 routesGenerated = 0
                 while routesGenerated < fanOut and len(nextLevelRoutes) > 0:
+                    # Create route from server to client span
```

Comment:
shouldn't the route be from client to server?

Reply:
Or you meant internal span?

Reply:
I believe it's from the same service's server span to its client span which will talk with another server